src/scene_battle_rpg2k3.cpp

```diff
@@ -760,13 +748,47 @@ void Scene_Battle_Rpg2k3::ProcessActions() {
 		return;
 	}
 
+	const bool wait = !CheckWait();
+
+	if (wait) {
+		return;
+	}
+
 	if (!battle_actions.empty()) {
 		auto* battler = battle_actions.front();
 		if (!battle_action_pending) {
 			// If we will start a new battle action, first check for state changes
 			// such as death, paralyze, confuse, etc..
 			PrepareBattleAction(battler);
+			auto* action = battler->GetBattleAlgorithm().get();
+
+			if (action->GetSource()->GetType() == Game_Battler::Type_Enemy && combo_repeat == 1) {
+				std::string notification = "";
+
+				if (action->GetType() == Game_BattleAlgorithm::Type::Normal && action->GetRepeat() == 2) {
+					notification = "Double Attack";
```

Comment:
Embed this logic into `Normal::GetStartMessage()`

Reply:
Changed.